.github/workflows/runnable_cxx.yml

```diff
@@ -176,7 +176,7 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ${{ github.workspace }}/clang+llvm-${{ matrix.cxx-version }}-${{ matrix.arch }}/
-        key: ${{ matrix.cxx-version }}-${{ matrix.arch }}
+        key: ${{ matrix.cxx-version }}-${{ matrix.arch }}-1
```

Comment:
Perhaps add a comment that the keep needs to be changed each time a change is made to the script below?

Reply:
There's not really a rule on how to change this tho. You might want to add a dimension to the key for example.
And next change we might remove the dash, or bump it to 42, there's really no semantic meaning tied to it.